From ce8bb5d37a22c1475b0162584db3e49507212f20 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Thu, 11 Jun 2020 07:03:58 +0300
Subject: [PATCH 014/140] Revert "Fix other lights when backlight uses
 SurfaceController API"

This reverts commit e952e58aabea968dc6dde2f70f6cdc593ff0a442.
---
 services/core/java/com/android/server/lights/LightsService.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/services/core/java/com/android/server/lights/LightsService.java b/services/core/java/com/android/server/lights/LightsService.java
index 0a308846181..5e82ae896fc 100644
--- a/services/core/java/com/android/server/lights/LightsService.java
+++ b/services/core/java/com/android/server/lights/LightsService.java
@@ -76,7 +76,6 @@ public class LightsService extends SystemService {
                 // right now we just fall back to the old path through Lights brightessMode is
                 // anything but USER or the device shouldBeInLowPersistenceMode().
                 if (brightnessMode == BRIGHTNESS_MODE_USER && !shouldBeInLowPersistenceMode()
-                        && mId == LightsManager.LIGHT_ID_BACKLIGHT
                         && mSurfaceControlMaximumBrightness == 255) {
                     // TODO: the last check should be mSurfaceControlMaximumBrightness != 0; the
                     // reason we enforce 255 right now is to stay consistent with the old path. In
-- 
2.25.1

